app/src/main/java/org/p2p/wallet/striga/signup/ui/StrigaSignUpFirstStepPresenter.kt

```diff
@@ -36,17 +52,27 @@ class StrigaSignUpFirstStepPresenter(
         view?.setButtonIsEnabled(isEnabled = true)
     }
 
-    override fun onCountryOfBirthChanged(selectedCountry: Country) {
-        setCachedData(StrigaSignupDataType.COUNTRY_OF_BIRTH, selectedCountry.codeAlpha3)
+    override fun onCountryChanged(newCountry: CountryCode) {
```

Comment:
and why onCountryOfBirthChanged is not ok? :)

Reply:
I don't know x D